cloud/fabfile.py

```diff
@@ -121,9 +117,9 @@ def prepdeploy():
     with lcd('/tmp/deploydir'):
         local('zip endagaweb_%s appspec.yml endagaweb_all.deb scripts/*'
               % (pkg_version))
-        local('aws s3 cp endagaweb_%s.zip s3://endagaweb-deployment/' % pkg_version)
+        local('aws s3 cp endagaweb_%s.zip s3://endagaweb-deployment-aricent/' % pkg_version)
```

Comment:
per earlier comments, I don't think the addition of the `-aricent` suffix is necessary, since names are not required to be globally unique.